File: src/main/java/com/xu/music/player/hander/MusicPlayerError.java
package com.xu.music.player.hander;

/**
 * 播放异常
 *
 * @since 2024年6月4日19点07分
 * @version swt-java/v1.0.0
 */
public class MusicPlayerError extends RuntimeException {

    public MusicPlayerError(String msg) {
        super(msg);
    }

    public MusicPlayerError(Throwable cause) {
        super(cause);
    }

    public MusicPlayerError(String msg, Throwable cause) {
        super(msg, cause);
    }

}

File: src/main/kotlin/com/xu/music/player/hander/MusicPlayerError.kt
package com.xu.music.player.hander

/**
 * 播放异常
 *
 * @date 2024年6月4日19点07分
 * @since SWT-V1.0.0.0
 */
class MusicPlayerError : RuntimeException {

    constructor(msg: String?) : super(msg)

    constructor(cause: Throwable?) : super(cause)

    constructor(msg: String?, cause: Throwable?) : super(msg, cause)

}
